apps/lndhub/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+        redis:
+              container_name: redis
+              image: redis:6.2-alpine
+              command: redis-server --requirepass moneyprintergobrrr
+              restart: on-failure
+              stop_grace_period: 5m
+              stop_signal: SIGINT
+              ports:
+                  - "$APP_LNDHUB_REDIS_PORT:$APP_LNDHUB_REDIS_PORT"
```

Comment:
No need to expose the Redis port here, as it's only used by the other container.